torch/utils/collect_env.py

```diff
@@ -111,15 +118,19 @@ def get_running_cuda_version(run_lambda):
 
 def get_cudnn_version(run_lambda):
     """This will return a list of libcudnn.so; it's hard to tell which one is being used"""
-    if get_platform() == 'win32':
+    platform = get_platform()
+    if platform == 'win32':
         cudnn_cmd = 'where /R "%CUDA_PATH%\\bin" cudnn*.dll'
+    elif platform == 'darwin':
+        cudnn_cmd = 'ls /usr/local/cuda/lib/libcudnn*'
```

Comment:
on darwin, I believe `/usr/local/cuda` isn't the right path. If we install CUDA on darwin, it gets installed under Developer? (or something like that)

Reply:
See [Install](https://docs.nvidia.com/cuda/cuda-installation-guide-mac-os-x/index.html#install) section of the NVIDIA CUDA Installation Guide for Mac OS X:
```
CUDA Driver: This will install /Library/Frameworks/CUDA.framework
and the UNIX-compatibility stub /usr/local/cuda/lib/libcuda.dylib
that refers to it.
```
So, `/usr/local/cuda/` should be fine provided that users follow installation instructions.
If they don't, we should not guess what they choose to do as there are too many ways to ignore the instructions.